apps/paf/fields/index.js

```diff
@@ -1327,7 +1328,8 @@ module.exports = {
   },
   'about-you-dob': dateComponent('about-you-dob', {
     isPageHeading: true,
-    mixin: 'input-date'
+    mixin: 'input-date',
+    validate: ['required', 'before', after1900Validator]
```

Comment:
date of birth is not a mandatory field so the 'required' validation should be removed.

Reply:
"required' validation removed.